filebeat/README.md

```diff
@@ -16,12 +16,12 @@ This Helm chart is a lightweight way to configure and run our official
   - [Install development version using master branch](#install-development-version-using-master-branch)
 - [Upgrading](#upgrading)
 - [Usage notes](#usage-notes)
-- [Configuration](#configuration)
```

Comment:
Tghis shouldn't be removed from the TOC. It seems this header was removed by mistake in a previous PR.